src/Components/Login/BoxLogin.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+
+class BoxLogin extends React.Component {
+  render(props) {
```

Comment:
```suggestion
  render() {
```
이 부분은 Render props라는 걸 사용할 때 써주시면 되는데 지금은 아닌 것 같습니다!

Reply:
넵 수정했습니다 :)